### ISSUE TITLE
different batchsize different loss value

### ISSUE THREAD

**Issue by AFOXCOW** [umbertogriffo/focal-loss-keras#5]
the keras custom loss function input shape require (batchsize,outputshape) or (outputshape,batchsize), but your code is calculate the loss of one output picture.
thus leading the loss value vary from the batchsize.
the first is calculated with batchsize=1
the second is calculated with batchsize=10
![a](https://user-images.githubusercontent.com/25144365/57577208-c03a1000-74a4-11e9-9a53-2260885c0477.png)

**Comment by decewei** [comment]
simple fix: K.mean instead of K.sum

**Comment by AFOXCOW** [comment]
Thanks a lot. Thats what I have done. 
But can you update the code for future readers.
Thanks again.

**Comment by Raazzta** [comment]
hi @AFOXCOW , @celinew1221 , in which line should we fix it?

**Comment by celinew1221** [comment]
I submitted a pull request regarding this.

**Comment by decewei** [comment]
> hi @AFOXCOW , @celinew1221 , in which line should we fix it?

I submitted a pull request regarding this.

**Comment by umbertogriffo** [comment]
Many Thanks @celinew1221 
Let's merge.
